packages/labs/src/components/timezone-select/_timezone-select.scss

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+@import "~@blueprintjs/core/src/common/variables";
+@import "~@blueprintjs/core/src/components/forms/common";
+@import "~@blueprintjs/core/src/components/tag/common";
+
+.pt-timezone-select-target {
+  &.pt-timezone-select-target-placeholder {
+    color: $pt-text-color-muted;
```

Comment:
is there a good way to indicate that a button is in a placeholder state? i did this, but didn't feel good about it.

Reply:
talked with @llorca about this, and we thought the best thing to do was to create a separate pr that adds "pt-placeholder" modifier to buttons that would have this effect - to mimic `input`'s `::placeholder`.

Reply:
https://github.com/palantir/blueprint/pull/1599